app/core/search.py

```diff
@@ -219,7 +219,13 @@ def process_result_into_csv(
                     "Document Title": document_title,
                     "Document URL": f"{url_base}/{document.slugs[-1].name}",
                     "Document Content URL": document_content,
-                    "Document Type": document.document_type,
+                    "Document Type": (
```

Comment:
Looks like a bug - as this can return "None" but document_type must be a string (pydantic)